lib/util.js

```diff
@@ -68,6 +68,7 @@ const types = require('internal/util/types');
 const {
   deprecate,
   getSystemErrorName: internalErrorName,
+  getSystemErrorMap,
```

Comment:
nit: keep ASCII order
```suggestion
  getSystemErrorMap,
  getSystemErrorName: internalErrorName,
```